app/controllers/merchant/items_controller.rb

```diff
@@ -2,7 +2,7 @@ class Merchant::ItemsController < ApplicationController
   before_action :require_merchant
 
   def index
-    @items = current_user.merchants[0].items
+    @merchant = current_user.merchants[0]
```

Comment:
merchant is a better object to pass than items and requires less view logic

Reply:
I agree!